roles/confluent.control_center/templates/override.conf.j2

```diff
@@ -1,6 +1,10 @@
 [Service]
 {% for key, value in control_center_service_overrides.items() %}
 {% if value %}
+{% if key =='ExecStart' %}
+# If there is an ExecStart override then we need to clear ExecStart first
```

Comment:
Can you explain this? The resulting override.conf would have 
```
ExecStart=
ExecStart=<something>
```
Why?

Reply:
That's the way systemd overrides work because I believe `ExecStart` is a list. According to https://www.freedesktop.org/software/systemd/man/systemd.unit.html

> Note that for drop-in files, if one wants to remove entries from a setting that is parsed as a list (and is not a dependency), such as AssertPathExists= (or e.g. ExecStart= in service units), one needs to first clear the list before re-adding all entries except the one that is to be removed. Dependencies (After=, etc.) cannot be reset to an empty list, so dependencies can only be added in drop-ins. If you want to remove dependencies, you have to override the entire unit.

Reply:
ah! Thanks for teaching me that! I was unaware it was a list/ you need to clear lists this way :)! 

I wonder if there is a way to make the comment more intuitive? maybe simply adding the work "list" would help
```suggestion
# If there is an ExecStart override then we need to clear ExecStart list first
```

wdyt? the goal being someone sees this code and wouldnt feel its necessary to look up why

Reply:
Done. "If there is an ExecStart override then we need to clear the ExecStart list first"